examples/tutorials/pipeline_end_to_end_tutorial.ipynb

```diff
@@ -0,0 +1,1472 @@
+       "      <td>3.0</td>\n",
+       "      <td>5.2</td>\n",
+       "      <td>2.0</td>\n",
+       "      <td>Virginica</td>\n",
+       "      <td>red</td>\n",
+       "      <td>2</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>148</th>\n",
+       "      <td>6.2</td>\n",
+       "      <td>3.4</td>\n",
+       "      <td>5.4</td>\n",
+       "      <td>2.3</td>\n",
+       "      <td>Virginica</td>\n",
+       "      <td>red</td>\n",
+       "      <td>2</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>149</th>\n",
+       "      <td>5.9</td>\n",
+       "      <td>3.0</td>\n",
+       "      <td>5.1</td>\n",
+       "      <td>1.8</td>\n",
+       "      <td>Virginica</td>\n",
+       "      <td>red</td>\n",
+       "      <td>2</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "<p>150 rows × 7 columns</p>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "     sepal.length  sepal.width  petal.length  petal.width    variety  \\\n",
+       "0             5.1          3.5           1.4          0.2     Setosa   \n",
+       "1             4.9          3.0           1.4          0.2     Setosa   \n",
+       "2             4.7          3.2           1.3          0.2     Setosa   \n",
+       "3             4.6          3.1           1.5          0.2     Setosa   \n",
+       "4             5.0          3.6           1.4          0.2     Setosa   \n",
+       "..            ...          ...           ...          ...        ...   \n",
+       "145           6.7          3.0           5.2          2.3  Virginica   \n",
+       "146           6.3          2.5           5.0          1.9  Virginica   \n",
+       "147           6.5          3.0           5.2          2.0  Virginica   \n",
+       "148           6.2          3.4           5.4          2.3  Virginica   \n",
+       "149           5.9          3.0           5.1          1.8  Virginica   \n",
+       "\n",
+       "    variety_color  variety_num  \n",
+       "0           green            0  \n",
+       "1           green            0  \n",
+       "2           green            0  \n",
+       "3           green            0  \n",
+       "4           green            0  \n",
+       "..            ...          ...  \n",
+       "145           red            2  \n",
+       "146           red            2  \n",
+       "147           red            2  \n",
+       "148           red            2  \n",
+       "149           red            2  \n",
+       "\n",
+       "[150 rows x 7 columns]"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# View data\n",
+    "df"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8a2eddfc",
+   "metadata": {},
+   "source": [
+    "Let's store the pre-processed data as an artifact since we may want to reuse/recompute it later."
```

Comment:
Assuming user knows what an artifact is, maybe have a link here or call out a prerequisites section before.

Reply:
Good point. Will put some notes giving a high-level explanation of what LineaPy artifact is and how it differs; and will provide a link to our RTD for more info.